daffodil-core/src/main/scala/org/apache/daffodil/dsom/ModelGroup.scala

```diff
@@ -153,14 +153,7 @@ abstract class ModelGroup
   final override def isRequiredOrComputed = true
   final override def isArray = false
 
-  private def prettyIndex = LV('prettyIndex) {
-    myPeers.map { peers =>
-      {
-        if (peers.length == 1) "" // no index expression if we are the only one
-        else "[" + (peers.indexOf(this) + 1) + "]" // 1-based indexing in XML/XSD
-      }
-    }.getOrElse("")
-  }.value
+  private def prettyIndex = "[" + index + "]" // 1-based indexing in XML/XSD
```

Comment:
Looks like this change caused some CLI tests to fail. I'd guess because we now always output "[X]" even if there are no peers. I think this change is fine since this is more a debug thing, so always outputting the index doesn't matter, but we should update the CLI tests so they all pass. To runt he CLI tests, you can run  ``sbt it:test``.

Reply:
Fixed this issue by having it look for seq[1] instead of just seq.